### PR TITLE
doc: Descriptions of input and output variables

### PR DESCRIPTION
**Pull request by rubenVB01**
## Description

- Added descriptions for input variables in variables.tf file
- Added descriptions for output variables in outputs.tf file

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -1,16 +1,19 @@
 # mongodbatlas_project
 
 output "project_id" {
+  description = "The project id."
```
Comment (EspenAlbert) [inline outputs.tf:4]: maybe specify an example, `The project id (e.g., "65def6ce0f722a1507105bb5")`

File: variables.tf [hunk 2]
```diff
@@ -1,28 +1,39 @@
 # mongodbatlas_project
 
 variable "org_id" {
+  description = "The ID of the organization you want to create the project within."
```
Comment (EspenAlbert) [inline variables.tf:4]: Same as above: https://github.com/terraform-mongodbatlas-modules/terraform-mongodbatlas-atlas-basic/pull/4/files#r1684214220

File: variables.tf [hunk 3]
```diff
@@ -41,26 +52,31 @@ variable "database_users" {
...
 }
 
 variable "backing_provider_name" {
+  description = "Cloud service provider on which you provision the host for a multi-tenant cluster."
   type    = string
   default = "AWS"
 }
 
 variable "region_name" {
+  description = "Physical location of your MongoDB cluster."
```
Comment (EspenAlbert) [inline variables.tf:73]: Maybe add a link to the MongoDB documentation: https://www.mongodb.com/docs/atlas/cloud-providers-regions/
Or alternatively, an example

**Review by EspenAlbert — APPROVED** [review]
Looks good. Only some minor comments

File: outputs.tf [hunk 4]
```diff
@@ -1,16 +1,19 @@
 # mongodbatlas_project
 
 output "project_id" {
+  description = "The project id (e.g., 65def6ce0f722a1507105bb5)."
   value = mongodbatlas_project.project.id
 }
 
 output "creation_timestamp" {
+  description = "The ISO-8601-formatted timestamp of when Atlas created the project."
```
Comment (lantoli) [inline outputs.tf:9]: some example?

File: variables.tf [hunk 5]
```diff
@@ -41,26 +52,31 @@ variable "database_users" {
 # mongodbatlas_advanced_cluster
 
 variable "cluster_name" {
+  description = "Name of the cluster as it appears in Atlas."
   type    = string
   default = "Cluster0"
 }
 
 variable "provider_name" {
+  description = "Cloud service provider on which the servers are provisioned. Possible values are AWS, GCP, AZUER or TENANT."
```
Comment (lantoli) [inline variables.tf:61]:
```suggestion
  description = "Cloud service provider on which the servers are provisioned. Possible values are AWS, GCP, AZURE or TENANT."
```

should we explain TENANT?

**Review by lantoli — APPROVED** [review]

**Review by AgustinBettati — APPROVED** [review]
LGTM, as mentioned in the daily lets make sure to have docs review the PR as these descriptions will be the main source for autogenerated documentation in the registry.

**Comment by rubenVB01** [comment]
> LGTM, as mentioned in the daily lets make sure to have docs review the PR as these descriptions will be the main source for autogenerated documentation in the registry.

Perfect, sent them a message through slack

File: variables.tf (hunk 5 above)
Comment (rubenVB01) [reply variables.tf:61]: Wanted to follow the same convention as AWS, Oracle, etc. where they put as description a brief sentence. I would suggest explaining it in the global README.md

File: variables.tf [hunk 6]
```diff
@@ -41,26 +52,31 @@ variable "database_users" {
...
 variable "region_name" {
-  type    = string
-  default = "US_EAST_1"
+  description = "Physical location of your MongoDB cluster (more information on https://www.mongodb.com/docs/atlas/cloud-providers-regions/)."
+  type        = string
+  default     = "US_EAST_1"
 }
 
 variable "electable_specs" {
+  description = "Hardware specifications for electable nodes in the region."
```
Comment (AgustinBettati) [inline variables.tf:79]: Should we mention that if provider is TENANT instance size should be M5 or lower? ([reference documentation](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Clusters/operation/createCluster)).

**Review by carriecwk — COMMENTED** [review]
A couple comments!

File: variables.tf (hunk 6 above)
Comment (rubenVB01) [reply variables.tf:79]: I tried to follow AWS, Oracle, etc. convention where they only use a brief sentence to explain the variable. I would mention it in the global readme

**Pull request merged**